### PR TITLE
Sara-Arash /Markets

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

add markets page

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

**Comment by arash-fs** [comment]
@mustofa-binary I already compress video now it's around 5MB. about the background I'm going to remove it :+1:

File: src/pages/markets/_available-trades.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+const Card = ({ name, active_tab, onTabChange }) => {
+    return (
+        <CardContainer name={name} active_tab={active_tab} onClick={() => onTabChange(name)}>
+            <Flex height="fit-content" p="2.71rem 0 0 3.2rem" jc="flex-start" ai="center">
+                {active_tab === 'Margin' && <MarginIcon />}
+                {active_tab === 'Options' && <OptionsIcon />}
+                {active_tab === 'Multipliers' && <MultipliersIcon />}
+                <Header as="h4" width="auto">
+                    {localize(name)}
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/_available-trades.js:94]: resolve it :+1:

File: src/pages/markets/_available-trades.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+                <Container
+                    direction="column"
+                    style={{
+                        marginTop: '2.8rem',
+                    }}
+                >
+                    <CardWrapper position="relative">
+                        {Margin && (
+                            <Card
+                                name="Margin"
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/_available-trades.js:170]: `name` is not using for functional purposes. it just pasess to the child to render it. I hardcoded to rendering the cards here. like this: if (the active tab  === `Margin`) then render `<Margin />` .
so the `name` hasn't any effect on this.

File: src/pages/markets/_hero.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%) scale(2);
+    width: 100%;
+    height: 100%;
+`
+export const Hero = () => {
+    return (
+        <BackgroundWrapper>
+            {/* TODO: we need to replace this DIV with an video later */}
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/_hero.js:35]: done

File: src/pages/markets/_learn-more.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+    @media ${device.laptopM} {
+        padding: 8rem 14rem;
+    }
+    @media ${device.tablet} {
+        padding: 8rem 4rem;
+    }
+`
+
+const AccordionWrapper = styled.div`
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/_learn-more.js:21]: we are going to remove this one, we don't have it based on the new design.

File: src/pages/markets/_what-trade.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    @media ${device.tablet} {
+        * {
+            margin-top: 1rem;
+            margin-right: 0;
+        }
+    }
+`
+export const WhatTrade = ({ header, sub_header }) => {
+    // const handleScrollToElement = () => {
+    //     learn_more_section.current.scrollIntoView()
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/_what-trade.js:37]: done

File: src/pages/markets/commodities/_commodities.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+import FriendlySupport from 'images/svg/friendly-support.svg'
+import ResponsiveWebsite from 'images/svg/responsive-website.svg'
+import NoCommission from 'images/svg/no-commission.svg'
+import MetalAndOil from 'images/svg/precious-metals-and-oils.svg'
+import VariableSpreads from 'images/svg/variable-spreads.svg'
+
+// const what_trade_sub_header = [
+//     {
+//         text: localize(
+//             'Commodities are resources that are grown or extracted from the ground, such as silver, gold, and oil.',
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/commodities/_commodities.js:19]: remove it

File: src/pages/markets/commodities/_commodities.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+            <WhyTrade
+                header={localize('Why trade commodities on Deriv')}
+                text={localize(
+                    'Speculate on the price movements of silver, gold, oil and more. Profit from the price difference when the market moves in the direction that you have predicted.',
+                )}
+            >
+                <div
+                    text={localize('Competitive fixed and variable spreads')}
+                    icon={<VariableSpreads />}
+                ></div>
```
[the remark this answers is not in the document]
Comment (arash-fs) [reply src/pages/markets/commodities/_commodities.js:44]: done

**Comment by ashkanx** [comment]
#### 📝Changed routes:


| [**/home/_markets**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/home/_markets) | [**/**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/) |
|:-:|:-:|
|<a href="https://deriv-com-git-fork-sara-fs-markets.binary.sx/home/_markets">Screenshot of /home/_markets</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/fV_1fqr3inTCws756Y9IPMirb6A_.png">(view full size)</a> |<a href="https://deriv-com-git-fork-sara-fs-markets.binary.sx/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/44evKjUPQZSDy3556nm2J3L_00E_.png">(view full size)</a> |



| [**…vg/_market-commodities**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-commodities) | [**…kets-svg/_market-forex**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-forex) |
|:-:|:-:|
|<a href="https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-commodities">Screenshot of /markets-svg/_market-commodities</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/A1xZsTdSjGlJxOkjtgfKxKWksLo_.png">(view full size)</a> |<a href="https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-forex">Screenshot of /markets-svg/_market-forex</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/obzvzW5RSiB4QTMBrURfvslcXqc_.png">(view full size)</a> |



| [**…kets-svg/_market-stock**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-stock) | [**…-svg/_market-synthetic**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-synthetic) |
|:-:|:-:|
|<a href="https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-stock">Screenshot of /markets-svg/_market-stock</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/K0VvlIrzAa5Cqx9_yUi2GRl4fMw_.png">(view full size)</a> |<a href="https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets-svg/_market-synthetic">Screenshot of /markets-svg/_market-synthetic</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/GC9xNqwOyR0K1E1QH6UKtosjhFI_.png">(view full size)</a> |



And 38 other routes:
- [**/markets/_available-options**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/_available-options)
- [**/markets/_available-trades**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/_available-trades)
- [**/markets/_hero**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/_hero)
- [**/markets/_other-markets**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/_other-markets)
- [**/markets/_what-trade**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/_what-trade)
- [**/markets/_why-trade**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/_why-trade)
- [**/markets/commodities/_commodities**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/commodities/_commodities)
- [**/markets/commodities/_digital-options**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/commodities/_digital-options)
- [**/markets/commodities/_margin**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/commodities/_margin)
- [**/markets/forex/_digital-options**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/forex/_digital-options)
- [**/markets/forex/_forex**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/forex/_forex)
- [**/markets/forex/_margin**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/forex/_margin)
- [**/markets/forex/_multipliers**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/forex/_multipliers)
- [**/markets/globe.mp4**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/globe.mp4)
- [**/markets/globe**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/globe)
- [**/markets**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets)
- [**/markets/stock/_digital-options**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/stock/_digital-options)
- [**/markets/stock/_margin**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/stock/_margin)
- [**/markets/stock/_stock-indices**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/stock/_stock-indices)
- [**/markets/sub-markets/_americas**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_americas)
- [**/markets/sub-markets/_asia-oceania**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_asia-oceania)
- [**/markets/sub-markets/_continuous-indices**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_continuous-indices)
- [**/markets/sub-markets/_crash-boom**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_crash-boom)
- [**/markets/sub-markets/_daily-reset-indices**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_daily-reset-indices)
- [**/markets/sub-markets/_energy**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_energy)
- [**/markets/sub-markets/_europe**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_europe)
- [**/markets/sub-markets/_exotic-pairs**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_exotic-pairs)
- [**/markets/sub-markets/_major-pairs**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_major-pairs)
- [**/markets/sub-markets/_metals**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_metals)
- [**/markets/sub-markets/_minor-pairs**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_minor-pairs)
- [**/markets/sub-markets/_smartfx**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_smartfx)
- [**/markets/sub-markets/_step-indices**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_step-indices)
- [**/markets/sub-markets/_submarkets**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_submarkets)
- [**/markets/sub-markets/_volatility-indices**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/sub-markets/_volatility-indices)
- [**/markets/synthetic/_digital-options**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/synthetic/_digital-options)
- [**/markets/synthetic/_margin**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/synthetic/_margin)
- [**/markets/synthetic/_multipliers**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/synthetic/_multipliers)
- [**/markets/synthetic/_synthetic-indices**](https://deriv-com-git-fork-sara-fs-markets.binary.sx/markets/synthetic/_synthetic-indices)

1 deleted route:
- **/home/_simple-steps**

Commit de8c8860466ceb55f9dc0e208376fa7c13a4b80b (https://deriv-com-git-fork-sara-fs-markets.binary.sx).

**Pull request merged**